coderd/database/migrations/000373_canonicalize_special_api_key_scopes.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Canonicalize special API key scopes to coder:* namespace
+-- Rename enum values: 'all' -> 'coder:all', 'application_connect' -> 'coder:application_connect'
+
+ALTER TYPE api_key_scope RENAME VALUE 'all' TO 'coder:all';
+ALTER TYPE api_key_scope RENAME VALUE 'application_connect' TO 'coder:application_connect';
```

Comment:
Ah, so renaming a type value doesn't require the same dance as adding a new enum type?